setup.py

```diff
@@ -156,7 +155,7 @@
     "rhoknp>=1.1.0,<1.3.1",
     "rjieba",
     "rouge-score!=0.0.7,!=0.0.8,!=0.1,!=0.1.1",
-    "ruff>=0.0.241,<=0.0.259",
+    "ruff>=0.1.5,<=0.2",
```

Comment:
Could you elaborate a bit on this choice?

Reply:
Latest version for better performance and unified api I'd say. While <=2 to have enough rolling in case they change things

Reply:
Oh, I misread  the numbers. Got it!